drivers/sensor/vcnl4040/vcnl4040_trigger.c

```diff
@@ -25,10 +26,11 @@ static void vcnl4040_gpio_callback(const struct device *dev,
 				   struct gpio_callback *cb,
 				   uint32_t pin_mask)
 {
+	const struct vcnl4040_config *config = dev->config;
```

Comment:
`dev` is a GPIO device here, not a sensor device

Reply:
Fixed